### PR TITLE
Release script

### PR DESCRIPTION
**Pull request by eggrobin**

### CODE REVIEW

File: documentation/Releasing.md [hunk 1]
```diff
@@ -9,32 +9,15 @@ names with the appropriate cases.
 - [ ] If using profile-guided optimization (PGO), make sure that the `PlayerTest.Benchmark` test contains the latest journal file name and has been merged as the last pull request.
 - [ ] Check that the release name, release date, and lunation number for the following update, used by the update reminder, are correctly set in the adapter.
 - [ ] Make sure that the project can be built and tested under Linux.
-- [ ] Create and push a tag for the current state of mockingbirdnest/master, where `"yyyyMMddHH"` is the date of the new moon in UT1:
+- [ ] Run the following command, where:
+  - `yyyyMMddHH` is the date of the new moon in UT1,
+  - `"<language>"` is the language code for the language in which `<Mathematician>` is written,
```
Comment (pleroy) [inline documentation/Releasing.md]: Why the double quoting around `language`?

Comment (pleroy) [reply documentation/Releasing.md]: What is the case of `Mathematician`?  It should be documented.

File: make_principia_release.ps1 [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+git fetch $remote
+
+if ($tag -in $(git tag)) {
+  write-error ("Tag $tag already exists.")
+}
+
+git checkout "$remote/master"
+git tag $tag -m $mathematician
+#git push $remote --tags REMOVE BEFORE FLIGHT: uncomment
```
Comment (pleroy) [inline make_principia_release.ps1]: Uncomment?

File: make_principia_release.ps1 [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+if (!(sls ([regex]::escape($primary_ksp_version)) -encoding Unicode `
+      ".\Release\GameData\Principia\ksp_plugin_adapter.dll")) {
+  write-error ("Configuration Release does not target $primary_ksp_version.")
+}
+
+foreach ($ksp_version in $compatibility_ksp_versions) {
+  if (!(sls ([regex]::escape($ksp_version)) -encoding Unicode `
+        (".\Release\$ksp_version Compatibility\GameData\Principia\" +
+             "ksp_plugin_adapter.dll"))) {
+    write-error ("Configuration Release KSP $ksp_version does not target "+
```
Comment (pleroy) [inline make_principia_release.ps1]: Space before `+`.

File: documentation/Releasing.md (hunk 1 above)
Comment (eggrobin) [reply documentation/Releasing.md]: It is (still) documented above.

**Pull request merged**